### PR TITLE
Lock files for 2.3.1.snapshot1

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -434,7 +431,7 @@ GEM
       logstash-filter-json
       logstash-input-generator
       logstash-output-file
-    logstash-output-elasticsearch (2.5.5-java)
+    logstash-output-elasticsearch (2.6.0-java)
```
Comment (jsvd) [inline Gemfile.jruby-1.9.lock]: maybe not bump this minor version?

**Comment by andrewvc** [comment]
LGTM

File: Gemfile.jruby-1.9.lock (hunk 1 above)
Comment (ph) [inline Gemfile.jruby-1.9.lock]: I reverted this change.

**Comment by untergeek** [comment]
LGTM™

**Pull request closed without merging**